test/integration.js

```diff
@@ -28,24 +29,34 @@ All files  |    91.18 |    88.89 |        0 |    91.18 |                   |
     const { output } = spawnSync(nodePath, [
       c8Path,
       '--exclude="test/*.js"',
+      '--clean=false',
       nodePath,
       require.resolve('./fixtures/multiple-spawn')
     ])
     output.toString('utf8').should.include(`
--------------------|----------|----------|----------|----------|-------------------|
-File               |  % Stmts | % Branch |  % Funcs |  % Lines | Uncovered Line #s |
--------------------|----------|----------|----------|----------|-------------------|
-All files          |      100 |    77.78 |      100 |      100 |                   |
- multiple-spawn.js |      100 |      100 |      100 |      100 |                   |
- subprocess.js     |      100 |    71.43 |      100 |      100 |              9,13 |
--------------------|----------|----------|----------|----------|-------------------|`)
+--------------------|----------|----------|----------|----------|-------------------|
+File                |  % Stmts | % Branch |  % Funcs |  % Lines | Uncovered Line #s |
+--------------------|----------|----------|----------|----------|-------------------|
+All files           |    94.12 |    70.59 |        0 |    94.12 |                   |
+ bin                |    83.72 |    57.14 |      100 |    83.72 |                   |
+  c8.js             |    83.72 |    57.14 |      100 |    83.72 |... 22,40,41,42,43 |
+ lib                |    96.41 |    65.38 |      100 |    96.41 |                   |
+  parse-args.js     |    97.47 |    44.44 |      100 |    97.47 |             55,56 |
+  report.js         |    95.45 |    76.47 |      100 |    95.45 |       51,52,53,54 |
+ test/fixtures      |    95.16 |    83.33 |        0 |    95.16 |                   |
```

Comment:
Is this supposed to be instrumented?

Reply:
yes, this allows me to test that various edge-cases are covered, e.g., subprocesses, JavaScript programs that exit in a weird way.

Once the actual coverage is output for the application itself, everything in `test` is removed from the report.